source/nodejs/adaptivecards-fabric/src/utils/shared.tsx

```diff
@@ -32,20 +32,25 @@ export abstract class ReactCardElementContainer extends AC.Container {
 
 export abstract class ReactInputElement extends AC.Input {
 
-    public valueInternal: any;
+    private _value: any;
     protected abstract renderReact(): JSX.Element;
 
     protected internalRender(): HTMLElement {
-        this.inputControlContainerElement.style.display = "block";
-        return sharedInternalRender(this.renderReact);
+        const element = sharedInternalRender(this.renderReact);
+        element.style.width = "100%";
+        return element;
     }
 
     protected handleChange = (event: React.FormEvent<HTMLInputElement | HTMLTextAreaElement>, newValue?: string) => {
-        this.valueInternal = newValue;
+        this._value = newValue;
+    }
+
+    get value(): any {
```

Comment:
`value` is supposed to be of type **string**. The underlying code expects that.

Reply:
I was following this example : https://docs.microsoft.com/en-us/adaptive-cards/sdk/rendering-cards/javascript/extensibility
here value is number. imo it makes more sense to have this as any, since not value are strings. For example input.number